src/utils/createChainedFunction.ts

```diff
@@ -0,0 +1,14 @@
+export const createChainedFunction = (...funcs: Function[]) => {
+  return funcs
+    .filter(f => f != null)
+    .reduce((accumulator: Function, func: Function) => {
```

Comment:
I would change `Function` to `() => any`

Reply:
Not the same. `Function` may have parameters as input (the case in my usage), furthermore, I am returning nothing as output.

I can change it to `(...args: any[]) => void`